pkg/apis/gke.cattle.io/v1/types.go

```diff
@@ -91,8 +91,8 @@ type GKENodePoolConfig struct {
 	Config            *GKENodeConfig          `json:"config,omitempty"`
 	InitialNodeCount  *int64                  `json:"initialNodeCount,omitempty"`
 	MaxPodsConstraint *int64                  `json:"maxPodsConstraint,omitempty"`
-	Name              *string                 `json:"name,omitempty"`
-	Version           *string                 `json:"version,omitempty"`
+	Name              *string                 `json:"name,omitempty" norman:"pointer"`
```

Comment:
Seems odd that name is in here

Reply:
It's the name of the node pool

Reply:
Talked offline, forgot this is for terraform, needs to be nil when we first register a cluster so that it can backfill. If its a blank string then it assumed we set that blank string on purpose.